intelmq/bots/outputs/redis/output.py

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8 -*-
+import sys
+import time
+import redis
+import intelmq.lib.utils as utils
+from intelmq.lib.bot import Bot
+
+
+class RedisOutputBot(Bot):
+
+    def init(self):
+        self.host = self.parameters.redis_server_ip
+        self.port = int(self.parameters.redis_server_port)
+        self.db = int(self.parameters.redis_db)
+        self.queue = self.parameters.redis_queue
+        self.password = self.parameters.redis_password
+        self.timeout = int(self.parameters.redis_timeout)
+
+        self.conn = redis.ConnectionPool(host=self.host, port=self.port, db=self.db)
+        self.connect()
+
+    def process(self):
+        event = self.receive_message()
+        
+        if event is None:
+           self.acknowledge_message()
+           return
+
+        while True:
```

Comment:
As I said, please remove the loop. We do not need to implement error handling multiple times.